app/src/main/res/values/strings.xml

```diff
@@ -48,5 +48,6 @@
 
     <!-- Accessibility -->
     <string name="a11y_plant_item_image" translation_description="Accessibility text for each of the plant images shown in the 'My garden' and 'Plant list' screens.">Picture of plant</string>
+    <string name="plant_date_header">Planted</string>
```

Comment:
move this to section above, probably under `garden_empty`.  Also add translation description